.github/workflows/check-samples.yml

```diff
@@ -54,16 +54,61 @@ jobs:
         run: |-
           sbt "publishLocal; publishM2"
 
+      - name: Test akka-quickstart-java
+        run: |-
+          cd samples/akka-quickstart-java
+          mvn test -nsu -ntp -Dakka.version=`cat ~/.version`
+
       - name: Test akka-quickstart-scala
         run: |-
           cd samples/akka-quickstart-scala
           sbt test -Dakka.version=`cat ~/.version`
 
-      - name: Test akka-quickstart-java
+      - name: Test akka-sample-cluster-java
         run: |-
-          cd samples/akka-quickstart-java
+          cd samples/akka-sample-cluster-java
+          mvn test -nsu -ntp -Dakka.version=`cat ~/.version`
+
+      - name: Test akka-sample-cluster-scala
+        run: |-
+          cd samples/akka-sample-cluster-scala
+          sbt test -Dakka.version=`cat ~/.version`
+
+      - name: Test akka-sample-distributed-data-java
+        run: |-
+          cd samples/akka-sample-distributed-data-java
           mvn test -nsu -ntp -Dakka.version=`cat ~/.version`
 
+      - name: Test akka-sample-distributed-data-scala
+        run: |-
+          cd samples/akka-sample-distributed-data-scala
+          sbt test -Dakka.version=`cat ~/.version`
+
+      - name: Test akka-sample-fsm-java
+        run: |-
+          cd samples/akka-sample-fsm-java
+          mvn test -nsu -ntp -Dakka.version=`cat ~/.version`
+
+      - name: Test akka-sample-fsm-scala
+        run: |-
+          cd samples/akka-sample-fsm-scala
+          sbt test -Dakka.version=`cat ~/.version`
+
+      - name: Test akka-sample-sharding-java
+        run: |-
+          cd samples/akka-sample-sharding-java
+          mvn test -nsu -ntp -Dakka.version=`cat ~/.version`
+
+      - name: Test akka-sample-sharding-scala
+        run: |-
+          cd samples/akka-sample-sharding-scala
+          sbt test -Dakka.version=`cat ~/.version`
+
+      - name: Test akka-sample-kafka-to-sharding-scala
+        run: |-
+          cd samples/akka-sample-kafka-to-sharding-scala
+          sbt test -Dakka.version=`cat ~/.version`
+
```

Comment:
The following are not moved (and will not have a replacement):
* akka-sample-cluster-client-grpc-java
* akka-sample-cluster-client-grpc-scala
* akka-sample-cqrs-java
* akka-sample-cqrs-scala
* akka-sample-distributed-workers-scala
* akka-sample-persistence-dc-java
* akka-sample-persistence-dc-scala
* akka-sample-persistence-java
* akka-sample-persistence-scala